packages/trader/build/config.js

```diff
@@ -27,13 +27,33 @@ const htmlOutputConfig = () => ({
 const htmlInjectConfig = () => ({
     links: [
         {
-            path: 'public/images/favicons',
-            glob: '*',
-            globPath: path.resolve(__dirname, '../src/public/images/favicons'),
+            path: 'favicon.ico',
             attributes: {
                 rel: 'icon',
             },
         },
+        ...[
+            { name: 'favicon', rel: 'icon', size: '16' },
+            { name: 'favicon', rel: 'icon', size: '32' },
+            { name: 'favicon', rel: 'icon', size: '96' },
+            { name: 'favicon', rel: 'icon', size: '160' },
+            { name: 'favicon', rel: 'icon', size: '192' },
+            { name: 'apple-touch-icon', size: '57' },
+            { name: 'apple-touch-icon', size: '60' },
+            { name: 'apple-touch-icon', size: '72' },
+            { name: 'apple-touch-icon', size: '76' },
+            { name: 'apple-touch-icon', size: '114' },
+            { name: 'apple-touch-icon', size: '120' },
+            { name: 'apple-touch-icon', size: '144' },
+            { name: 'apple-touch-icon', size: '152' },
+            { name: 'apple-touch-icon', size: '180' },
+        ].map(({ name, rel, size }) => ({
```

Comment:
I didn't introduce a variable for this array of favicons data because core and trader are served separately and have their own independent configs and constants. Also, please note that this htmlInjectConfig() is not used anywhere in trader at the moment so i've changed it here just for consistency.